src/components/navBar/index.tsx

```diff
@@ -78,40 +78,44 @@ const NavBar = () => {
                 </ul>
             </div>
             <div className={styles.userProfile}>
-                {!isLoggedIn ? (
-                    <Link href={LOGIN_URL}>
-                        <button className={styles.signInLink}>
-                            Sign In With Github
+                {userData !== undefined ? (
+                    userData ? (
```

Comment:
if userData happens to be `{}` it will return true, make sure you're handling that case

Reply:
The userData will either be undefined or an Object with Data properties right ?
Can you please elaborate on this empty Object { } case, Thank you